src/summary/history/allergy-card-level-three.css

```diff
@@ -0,0 +1,56 @@
+.allergyHeader {
+  margin: 0rem 0rem 0.25rem 1rem;
+  color: var(--omrs-color-ink-medium-contrast);
+}
+
+.container {
+  justify-content: center;
+  display: flex;
+  background: var(--omrs-color-ink-white);
+  padding-left: 1rem;
+  flex-direction: column;
+  border: 0.0625rem solid var(--omrs-color-ink-low-contrast);
+  margin: 0rem 1rem 0.5rem 1rem;
+  padding: 1rem;
+  max-height: 250px;
+  flex-wrap: wrap;
+}
+
+.container div {
+  margin-bottom: 1rem;
+}
+
+.container div:last-child {
+  margin-bottom: 0rem;
+}
+
+.allergyCommentContainer {
+  display: flex;
+  background: "var(--omrs-color-bg-medium-contrast)";
+  margin: 0rem 1rem 4.5rem 1rem;
+}
+
+.allergyCommentTextArea {
+  outline: none;
+  width: 100%;
+}
+
+.buttonStyles {
+  display: flex;
+  background: white;
+  bottom: 0;
+  justify-content: center;
+  position: absolute;
```

Comment:
Why are you using absolute position here? Absolute is relative to the document (not the parent element) - is that what you intended?

Reply:
The is to style the buttons to appear at the bottom of the screen as per the designs. Its parent to it is the `Summary Card style`

Reply:
ok